awssdk/src/main/scala/implicits.scala

```diff
@@ -1,32 +1,19 @@
 package meteor
 
-import cats.effect._
+import cats.effect.Async
 import cats.implicits._
 import meteor.errors.DecoderError
 
-import java.util.concurrent.{CompletableFuture, CompletionException}
+import java.util.concurrent.CompletableFuture
 import java.util.{HashMap => jHashMap, Map => jMap}
 
 private[meteor] object implicits extends syntax {
   type FailureOr[U] = Either[DecoderError, U]
 
-  implicit class FromCompletableFuture[A](thunk: () => CompletableFuture[A]) {
```

Comment:
`Async` extends `MonadCancel`, and is cancelable. I looked at the internals of `Async.fromCompletableFuture` and the error handling code looks the same to me, so I think it's safe to replace this with `liftFuture` below, and it's more concise.